css/style.css

```diff
@@ -43,6 +43,199 @@ Developer @GA in NYC
 /*******************************************/
 
 body {
+  font-family: "Open Sans", helvetica; 
+  clear: both;
+  overflow: auto;
+}
+
+
+header {
+  background-color: #033048;
+}
+
+h1 {
+  float: left;
+  width: 30%;
+  font-size: 24px;
+  font-weight: bold;
+  padding-left: 30px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  color: white;
+}
+
+.headernav {
+  float: right;
+  width: 70%;
+  padding: 10px;
+}
+
+.headerlist {
+  list-style-type: none;
+  float: left;
+  width: 20%;
+  color: #f9e42e;
+  padding: 10px;
+}
+
+.rightside {
+  width: 70%;
+  padding-right: 10px;
+  padding-left: 10px;
+  padding-top: 20px;
+  padding-bottom: 15px;
+  overflow: auto;
+}
+
+main {
+  background-color: #f0efef;
+  overflow: auto;
+  clear: both;
+}
+
+.firstsection {
+  float: left;
+  width: 70%;
+}
+
+
+.secondsection {
+  float: left;
+  width: 70%;
+  background-color: ;
+}
+
+.title {
+  font-size: 40px;
+  font-weight: 800;
+  padding-top: 20px;
+  padding-bottom: 15px;
+  color: #033048;
+}
+
+aside {
+  float: right;
+  width: 30%;
+  padding-right: 10px;
+  clear: both;
+  overflow: auto;
+  display: inline;
+}
+
+p {
+  font-size: 24px;
+  padding: 10px;
+}
+
+.rightside p::first-letter {
+  font-size: 40px;
+}
+
+nav {
+  float: left;
+  width: 100%;
+  clear: both;
+}
+
+.About {
+  float: left;
+  width: 100%;
+  padding-top: 20px;
+  clear: both;
+}
+
+.ad {
+  float: left;
+  width: 100%;
+  clear: both;
+}
+
+.sidebarheader {
```

Comment:
Another thing we probably want to do with our class names is somehow make the break between words more obvious. So, instead of `.sidebarheader`, we might use `.sidebar-header` or `.sidebarHeader`. This just makes it a bit more readable.